astropy/modeling/core.py

```diff
@@ -69,7 +69,7 @@ def _model_oper(oper, **kwargs):
 
     # Perform an arithmetic operation on two models.
     return lambda left, right: _CompoundModelMeta._from_operator(oper,
-            left, right, **kwargs)
+                                                                 left, right, **kwargs)
```

Comment:
In the future, try to avoid whitespace changes in unrelated parts of the code for your PRs ... it makes it harder to review / less focused.

So the best solution is to have a good Python text editor (like PyCharm or emacs or ...) that helps you write correctly formatted code automatically (e.g. PyCharm adds yellow squiggles to any mis-formatted code as you type).

If you want to run autopep8 or some other code-formatting tool, it's best to just select your new code in the editor and run it on that part, instead of running it on the whole file you've worked on.